### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by gvodianytskyi**

### CODE REVIEW

File: src/components/product-page/product-page.js [hunk 1]
```diff
@@ -35,11 +36,26 @@ export default class ProductPage extends PureComponent {
...
-        <Text h3>${product.price}</Text>
+        <Text h1 style={styles.title}>{product.title}</Text>
+        <Swiper style={styles.slider}
+                loop
+                showsButtons
+                dot={<View style={styles.dot} />}
+                activeDot={<View style={styles.activeDot} />}
+                prevButton={<Text style={styles.controlButton}>‹</Text>}
+                nextButton={<Text style={styles.controlButton}>›</Text>}>
+          {product.images.map(item => {
```
Comment (httpEugene) [inline src/components/product-page/product-page.js]: I would move this to separate method

**Comment by sergeyamator** [comment]
Travis failed

File: src/components/product-page/product-page.js [hunk 2]
```diff
@@ -35,11 +36,26 @@ export default class ProductPage extends PureComponent {
 
   renderProduct(product) {
     return product ? (
+      <ScrollView>
```
Comment (sergeyamator) [inline src/components/product-page/product-page.js]: Will be better to use flatlist component instead scroll view

File: src/components/product-page/product-page.js [hunk 3]
```diff
@@ -33,13 +34,30 @@ export default class ProductPage extends PureComponent {
...
+        <View style={styles.container}>
+          <Text h1 style={styles.title}>{product.title}</Text>
+          <Swiper style={styles.slider}
+                  loop
+                  showsButtons
+                  dot={<View style={styles.dot} />}
+                  activeDot={<View style={styles.activeDot} />}
+                  prevButton={<Text style={styles.controlButton}>‹</Text>}
+                  nextButton={<Text style={styles.controlButton}>›</Text>}>
+            {product.images.map(this.renderSlide)}
```
Comment (httpEugene) [inline src/components/product-page/product-page.js]: you can pass this.renderSlide.url

File: src/components/product-page/product-page.js [hunk 4]
```diff
@@ -33,13 +34,30 @@ export default class ProductPage extends PureComponent {
...
-      <View style={styles.container}>
-        <Text h1>{product.title}</Text>
-        <Text>{product.description}</Text>
-        <Text h3>${product.price}</Text>
-      </View>
+      <ScrollView>
+        <View style={styles.container}>
+          <Text h1 style={styles.title}>{product.title}</Text>
+          <Swiper style={styles.slider}
+                  loop
```
Comment (sergeyamator) [inline src/components/product-page/product-page.js]: could you pretty props, the first should start from the next line

File: src/components/product-page/product-page.js [hunk 5]
```diff
@@ -33,13 +34,30 @@ export default class ProductPage extends PureComponent {
     this.props.getProductData();
   }
 
+  renderSlide(image) {
+    return (<View style={styles.slide} key={image.url}>
+      <Image style={styles.image} source={{ uri: image.url }}/>
+    </View>);
```
Comment (sergeyamator) [inline src/components/product-page/product-page.js]: How about this style
```
 return (
  <View style={styles.slide} key={image.url}>
    <Image style={styles.image} source={{ uri: image.url }}/>
  </View>
);
```

**Pull request merged**